src/tests.rs

```diff
@@ -17,26 +17,32 @@
 
 #![cfg(test)]
 
-use maidsafe_utilities::serialisation::serialise;
-use maidsafe_utilities::thread::RaiiThreadJoiner;
 use sodiumoxide::crypto;
 use sodiumoxide::crypto::hash::sha512;
 use std::iter;
 use std::sync::mpsc::{self, Sender, Receiver, TryRecvError};
 use std::thread;
 use std::time::Duration;
-use xor_name::XorName;
+use std::cmp::Ordering::{Less, Greater};
+use time;
+use rand;
+use itertools::Itertools;
 
+use xor_name;
+use xor_name::XorName;
+use kademlia_routing_table;
+use maidsafe_utilities::serialisation::serialise;
+use maidsafe_utilities::thread::RaiiThreadJoiner;
 use authority::Authority;
 use client::Client;
 use data::Data;
 use event::Event;
 use id::FullId;
-use messages::{RequestContent, ResponseContent, ResponseMessage};
+use messages::{RequestContent, ResponseContent, RequestMessage, ResponseMessage};
 use node::Node;
 use plain_data::PlainData;
 
-const GROUP_SIZE: usize = 8;
+const GROUP_SIZE: usize = kademlia_routing_table::GROUP_SIZE as usize;
```

Comment:
@afck I don't think we want to do those conversions, or in fact use them as `u8`'s, since AFAICT we don't currently. So 'usize`'s look fitting. @dirivine  It appears to be in the pipeline for the language to handle. More problematic is the tests time out in Appveyor.